src/GovUk.Education.ExploreEducationStatistics.Admin/Services/UserManagementService.cs

```diff
@@ -166,69 +174,74 @@ public async Task<UserViewModel> GetAsync(string userId)
             return null;
         }
 
-        public async Task<List<UserViewModel>> ListPendingAsync()
+        public async Task<Either<ActionResult, List<UserViewModel>>> ListPendingInvites()
         {
-            var pendingUsers = await _usersAndRolesDbContext.UserInvites.Where(u => u.Accepted == false)
-                .OrderBy(x => x.Email).Select(u => new UserViewModel
-                {
-                    Email = u.Email,
-                    Role = u.Role.Name
-                }).ToListAsync();
-
-            return pendingUsers;
+            return await _userService
+                .CheckCanManageAllUsers()
+                .OnSuccess(async () =>
+                    await _usersAndRolesDbContext.UserInvites.Where(u => u.Accepted == false)
+                        .OrderBy(x => x.Email).Select(u => new UserViewModel
+                        {
+                            Email = u.Email,
+                            Role = u.Role.Name
```

Comment:
I don't think there's any unit test for this yet. It should be possible via writing a unit test to demonstrate there's going to be a NullReferenceException possibly occuring here since there's a missing `.Include` on `u.Role`.

Reply:
I've added the Include. I'll create a separate subtask to add a unit test to catch this